py3-jupyter-console.yaml

```diff
@@ -0,0 +1,120 @@
+package:
+  name: py3-jupyter-console
+  version: 6.6.3
+  epoch: 1
+  description: A terminal-based console frontend for Jupyter kernels.
+  copyright:
+    - license: BSD-3-Clause
+  dependencies:
+    provider-priority: 0
```

Comment:
What does this setting do in this package?

Reply:
Other packages that provide `py3-blah-blah-blah` will take priority based on the version of Python (latest will always be resolved to). This package is empty so we set a priority of 0 so that it never gets installed

Reply:
ahhh TIL!